app/controllers/admin/companies_controller.rb

```diff
@@ -15,8 +15,24 @@ def destroy
 
   def update
     company = Company.find(params[:id])
+    company.update(update_params)
     company.update(status: params[:company][:status])
-    company.save
-    redirect_to admin_company_path(company.id)
+    if company.save
+      redirect_to admin_company_path(company.id)
+    else
+      flash[:errors] = "Please try again!"
+      render :edit
+    end
+  end
+
+  def edit
+    @company = Company.find(params[:id])
+  end
+
+  private
+
+  def update_params
+    params.require(:company).permit(:name, :organization, :title,
+                                 :email, :state, :city)
```

Comment:
Wouldn't we want the ability to edit a company's intention to hire, if they get more info and change their mind?  same goes for hire_count.